src/main_app/constants.js

```diff
@@ -5,6 +5,7 @@ export const PROCESS_CALENDLY_EVENT_INVITEE_ENDPOINT_URL = (
   process.env.REACT_APP_CALENDLY_EVENT_PROCESSOR_URL ||
   'http://localhost:5001/wedevelop-site/us-central1/processCalendlyEventInvitee'
 )
+export const PROCESS_STAFF_AUGMENTATION_FORM_SUBMIT = 'http://localhost:5001/wedevelop-site/us-central1/processStaffAugmentationFormSubmit'
```

Comment:
- Please use environment variables with a default value, as I did with the constant from the previous line.
- Also, the name of a constant should indicate what it contains. This name is an action, and actions are for functions. Try a different name such as `STAFF_AUGMENTATION_FORM_PROCESSOR_URL`
```suggestion
export const STAFF_AUGMENTATION_FORM_PROCESSOR_URL = (
  process.env.REACT_APP_STAFF_AUGMENTATION_FORM_PROCESSOR_URL ||
  'http://localhost:5001/wedevelop-site/us-central1/processStaffAugmentationFormSubmit'
)
```